docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -70,6 +86,61 @@ How to reset a macOS host's password using the disk encryption key:
 
+### Custom settings
+
+In Fleet you can enforce custom settings on your macOS hosts using configuration profiles.
+
+To enforce custom settings, first create configuration profiles with iMazing Profile editor and then add the profiles to Fleet. 
+
+#### Create a configuration profiles with iMazing Profile Creator
+
+How to create a configuration profile with iMazing Profile Creator:
+
+1. Download and install [iMazing Profile Creator](https://imazing.com/profile-editor).
+
+2. Open iMazing Profile Creator and select macOS in the top bar. Fleet only supports enforcing settings on macOS hosts.
+
+3. Find and choose the settings you'd like to enforce on your macOS hosts. Fleet recommends limiting the scope of the settings a single profile: only include settings from one tab in iMazing Profile Creator (ex. **Restrictions** tab). To enforce more settings, you can create and add additional profiles.
+
+4. In iMazing Profile Creator, select the **General** tab. Enter a descriptive name in the **Name** field. When you add this profile to Fleet, Fleet will display this name in the Fleet UI. 
+
+5. In your top menu bar select **File** > **Save As...** and save your configuration profile. Make sure the file is saved as .mobileconfig.
+
+#### Add configuration profiles to Fleet
+
+In Fleet, you can add configuration profiles using the Fleet UI or fleetctl command-line tool.
+
+The Fleet UI method is a good start if you're just getting familiar with Fleet.
+
+The fleetctl CLI method enables managing configuration profiles in a git repository. This way you can enforce code review and benefit from git's change history.
+
+Fleet UI:
+
+1. In the Fleet UI, head to the **Controls > macOS settings > Custom settings** page.
+
+2. Select **Upload** and choose your configuration profile. After your configuration profile is uploaded to Fleet, Fleet will apply the profile on your macOS hosts. The profile will be applied to new macOS hosts that enroll to Fleet.
+
+fleetctl CLI:
+
+1. Create a `config` YAML document if you don't have one already. Learn how [here](./configuration-files/README.md#organization-settings). This document is used to change settings in Fleet.
+
+> If you want to add configuration profiles to all macOS hosts on a specific team in Fleet, use the `team` YAML document. Learn how to create one [here](./configuration-files/README.md#teams).
+
+2. Add an `mdm.macos_settings.custom_settings` key to your YAML document. This key will hold an array of paths to your configuration profiles. See the below example `config` YAML document:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  macos_settings:
+    custom_settings:
+      - /path/to/configuration_profile_A.mobileconfig
+      - /path/to/configuration_profile_B.mobileconfig
+    ...
+```
+
+3. Run the `fleetctl apply -f <your-config-here>.yml` command to add the configuration profiles to Fleet. Note that this will override any configuration profiles added using the Fleet UI method.
```

Comment:
I think another line that's like "you can also do this for teams" (I'm pretty sure that's gonna be used out of the gate).

Reply:
Hey! I have a line about teams here: https://github.com/fleetdm/fleet/pull/10055/files#diff-bae896c6452d8ac6968dd79d7542caf1416c59bfda0393c9fe363ebe36e510c2R127

Do you think we should move it?

Reply:
Yeah cuz that section is for disk encryption right? hmmm could get repetitive. But I think it's worth mentioning here until we find a more abstracted way to present it.